collections/_members/olushesi-toheeb.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
This file and your profile photo look great, @OlushesiToheeb. Can we remove all of the other files from this PR?

Reply:
it was only two files that i changed

Reply:
maybe I'll just have to clone again.